src/pages/LoginPage.js

```diff
@@ -1,6 +1,9 @@
 import React, { useState, useEffect } from "react";
 import image from "../images/image-1.png";
 import { useNavigate } from "react-router-dom";
+import Button from "react-bootstrap/Button";
+import Container from "react-bootstrap/Container";
+import Form from "react-bootstrap/Form";
```

Comment:
`import` statements should be ordered by rank and alphabetically.
Rank:
1. First party, example: react
2. Third party, example: react-router-dom, react-bootstrap
3. Packages we wrote.